### PR TITLE
update mono usage in README

### PR DESCRIPTION
**Pull request by strogiyotec**
https://github.com/r2dbc/r2dbc-postgresql/issues/105

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@strogiyotec Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=r2dbc/r2dbc-postgresql&pullRequestId=106)!

[Click here](https://cla.pivotal.io/sync/reactor?repositoryId=r2dbc/r2dbc-postgresql&pullRequestId=106) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

**Comment by pivotal-issuemaster** [comment]
@strogiyotec Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/reactor?repositoryId=r2dbc/r2dbc-postgresql&pullRequestId=106)!

File: README.md [hunk 1]
```diff
@@ -111,12 +111,16 @@ INSERT INTO person (id, first_name, last_name) VALUES ($1, $2, $3)
 Parameters are referenced using the same identifiers when binding these:
 
 ```java
-connection
```
Comment (nebhale) [inline README.md]: I wouldn't change this directly (certainly not to a user of `subscribe()` which is almost never correct).  I think the key either needs to be solely the former name change, or _possibly_ a more complex example with a `flatMap()`.  I lean towards keeping this simpler (eliminating the outer layer where you get the connection from the `Mono`, but I can see an argument where it's much more representative of real usage.

Comment (strogiyotec) [reply README.md]: OK, I will try to provide a more complex example

Comment (strogiyotec) [reply README.md]: @nbenjamin 
Could you review it now , I think that it's much better to show how to get the connection from the Mono . I have used **map** function . What do you think about it ?

**Comment by strogiyotec** [comment]
@nebhale What did you decide about this ticket ?

File: README.md (hunk 1 above)
Comment (mp911de) [reply README.md]: Please remove `.block().blockFirst();` as calling these methods in a reactive context is considered an antipattern.

Comment (strogiyotec) [reply README.md]: @mp911de  I did it , could you review it now ?

**Comment by mp911de** [comment]
That's merged and polished now.

**Pull request closed without merging**